pyOCD/interface/hidapi_backend.py

```diff
@@ -107,3 +107,7 @@ def close(self):
         """
         logging.debug("closing interface")
         self.device.close()
+
+    def setPacketCount(self, count):
+        # No interface level restrictions on count
+        self.packet_count = count
```

Comment:
Ah that explains it.  I tried it on windows and it gives similar performance improvements.